nullaway/src/main/java/com/uber/nullaway/handlers/contract/ContractNullnessStoreInitializer.java

```diff
@@ -27,6 +27,8 @@
  */
 public class ContractNullnessStoreInitializer extends NullnessStoreInitializer {
 
+  static final String ANNOT_NAME = "org.jetbrains.annotations.Contract";
```

Comment:
Maybe this annotation name for Contract should be stored in a single place, rather than in two separate static fields?